mediapipe/pose_tracking_dll/BUILD

```diff
@@ -31,26 +31,19 @@ windows_dll_library(
         "//mediapipe/framework/formats:landmark_cc_proto",
         "//mediapipe/framework/formats:rect_cc_proto",
         "//mediapipe/framework/port:file_helpers",
-        "//mediapipe/framework/port:opencv_highgui",
-        "//mediapipe/framework/port:opencv_imgproc",
-        "//mediapipe/framework/port:opencv_video",
         "//mediapipe/framework/port:parse_text_proto",
         "//mediapipe/framework/port:status",
         "@com_google_absl//absl/flags:flag",
         "@com_google_absl//absl/flags:parse",
 
-        "//mediapipe/calculators/core:constant_side_packet_calculator",
-        "//mediapipe/calculators/core:packet_presence_calculator",
-        "//mediapipe/calculators/core:flow_limiter_calculator",
-        "//mediapipe/calculators/tflite:tflite_model_calculator",
-        "//mediapipe/calculators/util:local_file_contents_calculator",
         "//mediapipe/graphs/pose_tracking:pose_tracking_cpu_deps",
     ]
 )
 
 # **Implicitly link to face_mesh_lib.dll**
 cc_binary(
     name = "pose_tracking_cpu",
+	linkopts = ["/FORCE:MULTIPLE"],
```

Comment:
I forgot what was the cause of having duplicate symbols.

Reply:
I added a comment explaining the cause